### ISSUE TITLE
metx_Read: AddressSanitizer failed to allocate 0x100002000 (4294975488) bytes of LargeMmapAllocator

### ISSUE THREAD

**Issue by Beuc** [gpac/gpac#1393]
Hi,

- [X] I looked for a similar issue and couldn't find any.
- [X] I tried with the latest version of GPAC. Installers available at http://gpac.io/downloads/gpac-nightly-builds/
- [X] I give enough information for contributors to reproduce my issue (meaningful title, github labels, platform and compiler, command-line ...). I can share files anonymously with this dropbox: https://www.mediafire.com/filedrop/filedrop_hosted.php?drop=eec9e058a9486fe4e99c33021481d9e1826ca9dbc242a6cfaab0fe95da5e5d95

While checking #1337 with the same environment, I noted that the PoC crashed (while it should cover memory leaks).

It appears there's a malloc of UINT_MAX at
https://github.com/gpac/gpac/blob/7ec96e7eaf2dd69a888e944449f0059ac86999b5/src/isomedia/box_code_base.c#L8446
https://github.com/gpac/gpac/blob/7ec96e7eaf2dd69a888e944449f0059ac86999b5/src/isomedia/box_code_base.c#L8447
which fails.

This consistently occurs with all versions I tested from master back to 0.5.0.
Without ASAN this results in a segfault.

```
$ MP4Box -diso -out /dev/null ../CVE-2019-20171/POC-new-memory-leak
ptr->size: 7
size: 4294967295
==4700==ERROR: AddressSanitizer failed to allocate 0x100002000 (4294975488) bytes of LargeMmapAllocator (errno: 12)
==4700==Process memory map follows:
	0x000000400000-0x0000004bf000	/home/gpac/gpac_public-master/destdir/usr/local/bin/MP4Box
	0x0000006be000-0x0000006bf000	/home/gpac/gpac_public-master/destdir/usr/local/bin/MP4Box
	0x0000006bf000-0x000000769000	/home/gpac/gpac_public-master/destdir/usr/local/bin/MP4Box
	0x000000769000-0x00000076c000	
	0x00007fff7000-0x00008fff7000	
	0x00008fff7000-0x02008fff7000	
	0x02008fff7000-0x10007fff8000	
	0x600000000000-0x602000000000	
	0x602000000000-0x602000010000	
	0x602000010000-0x604000000000	
	0x604000000000-0x604000010000	
	0x604000010000-0x606000000000	
	0x606000000000-0x606000010000	
	0x606000010000-0x607000000000	
	0x607000000000-0x607000010000	
	0x607000010000-0x608000000000	
	0x608000000000-0x608000010000	
	0x608000010000-0x60b000000000	
	0x60b000000000-0x60b000010000	
	0x60b000010000-0x60c000000000	
	0x60c000000000-0x60c000010000	
	0x60c000010000-0x60f000000000	
	0x60f000000000-0x60f000010000	
	0x60f000010000-0x612000000000	
	0x612000000000-0x612000010000	
	0x612000010000-0x616000000000	
	0x616000000000-0x616000020000	
	0x616000020000-0x619000000000	
	0x619000000000-0x619000020000	
	0x619000020000-0x621000000000	
	0x621000000000-0x621000020000	
	0x621000020000-0x624000000000	
	0x624000000000-0x624000020000	
	0x624000020000-0x631000000000	
	0x631000000000-0x631000030000	
	0x631000030000-0x640000000000	
	0x640000000000-0x640000003000	
	0x7f7f6fc87000-0x7f7f723cb000	
	0x7f7f723cb000-0x7f7f723d0000	/usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
	0x7f7f723d0000-0x7f7f725cf000	/usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
	0x7f7f725cf000-0x7f7f725d0000	/usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
	0x7f7f725d0000-0x7f7f725d1000	/usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
	0x7f7f725d1000-0x7f7f725d3000	/usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
	0x7f7f725d3000-0x7f7f727d3000	/usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
	0x7f7f727d3000-0x7f7f727d4000	/usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
	0x7f7f727d4000-0x7f7f727d5000	/usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
	0x7f7f727d5000-0x7f7f727e5000	/usr/lib/x86_64-linux-gnu/libdrm.so.2.4.0
	0x7f7f727e5000-0x7f7f729e5000	/usr/lib/x86_64-linux-gnu/libdrm.so.2.4.0
	0x7f7f729e5000-0x7f7f729e6000	/usr/lib/x86_64-linux-gnu/libdrm.so.2.4.0
	0x7f7f729e6000-0x7f7f729e7000	/usr/lib/x86_64-linux-gnu/libdrm.so.2.4.0
	0x7f7f729e7000-0x7f7f729ec000	/usr/lib/x86_64-linux-gnu/libXxf86vm.so.1.0.0
	0x7f7f729ec000-0x7f7f72beb000	/usr/lib/x86_64-linux-gnu/libXxf86vm.so.1.0.0
	0x7f7f72beb000-0x7f7f72bec000	/usr/lib/x86_64-linux-gnu/libXxf86vm.so.1.0.0
	0x7f7f72bec000-0x7f7f72bed000	/usr/lib/x86_64-linux-gnu/libXxf86vm.so.1.0.0
	0x7f7f72bed000-0x7f7f72c0e000	/usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
	0x7f7f72c0e000-0x7f7f72e0d000	/usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
	0x7f7f72e0d000-0x7f7f72e0e000	/usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
	0x7f7f72e0e000-0x7f7f72e0f000	/usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
	0x7f7f72e0f000-0x7f7f72e13000	/usr/lib/x86_64-linux-gnu/libxcb-dri2.so.0.0.0
	0x7f7f72e13000-0x7f7f73012000	/usr/lib/x86_64-linux-gnu/libxcb-dri2.so.0.0.0
	0x7f7f73012000-0x7f7f73013000	/usr/lib/x86_64-linux-gnu/libxcb-dri2.so.0.0.0
	0x7f7f73013000-0x7f7f73014000	/usr/lib/x86_64-linux-gnu/libxcb-dri2.so.0.0.0
	0x7f7f73014000-0x7f7f7302b000	/usr/lib/x86_64-linux-gnu/libxcb-glx.so.0.0.0
	0x7f7f7302b000-0x7f7f7322a000	/usr/lib/x86_64-linux-gnu/libxcb-glx.so.0.0.0
	0x7f7f7322a000-0x7f7f7322c000	/usr/lib/x86_64-linux-gnu/libxcb-glx.so.0.0.0
	0x7f7f7322c000-0x7f7f7322d000	/usr/lib/x86_64-linux-gnu/libxcb-glx.so.0.0.0
	0x7f7f7322d000-0x7f7f73362000	/usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
	0x7f7f73362000-0x7f7f73562000	/usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
	0x7f7f73562000-0x7f7f73563000	/usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
	0x7f7f73563000-0x7f7f73567000	/usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
	0x7f7f73567000-0x7f7f73568000	/usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
	0x7f7f73568000-0x7f7f73767000	/usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
	0x7f7f73767000-0x7f7f73768000	/usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
	0x7f7f73768000-0x7f7f73769000	/usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
	0x7f7f73769000-0x7f7f7376e000	/usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
	0x7f7f7376e000-0x7f7f7396d000	/usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
	0x7f7f7396d000-0x7f7f7396e000	/usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
	0x7f7f7396e000-0x7f7f7396f000	/usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
	0x7f7f7396f000-0x7f7f73971000	/usr/lib/x86_64-linux-gnu/libXdamage.so.1.1.0
	0x7f7f73971000-0x7f7f73b70000	/usr/lib/x86_64-linux-gnu/libXdamage.so.1.1.0
	0x7f7f73b70000-0x7f7f73b71000	/usr/lib/x86_64-linux-gnu/libXdamage.so.1.1.0
	0x7f7f73b71000-0x7f7f73b72000	/usr/lib/x86_64-linux-gnu/libXdamage.so.1.1.0
	0x7f7f73b72000-0x7f7f73b83000	/usr/lib/x86_64-linux-gnu/libXext.so.6.4.0
	0x7f7f73b83000-0x7f7f73d82000	/usr/lib/x86_64-linux-gnu/libXext.so.6.4.0
	0x7f7f73d82000-0x7f7f73d83000	/usr/lib/x86_64-linux-gnu/libXext.so.6.4.0
	0x7f7f73d83000-0x7f7f73d84000	/usr/lib/x86_64-linux-gnu/libXext.so.6.4.0
	0x7f7f73d84000-0x7f7f73db0000	/usr/lib/x86_64-linux-gnu/libglapi.so.0.0.0
	0x7f7f73db0000-0x7f7f73faf000	/usr/lib/x86_64-linux-gnu/libglapi.so.0.0.0
	0x7f7f73faf000-0x7f7f73fb3000	/usr/lib/x86_64-linux-gnu/libglapi.so.0.0.0
	0x7f7f73fb3000-0x7f7f73fb4000	/usr/lib/x86_64-linux-gnu/libglapi.so.0.0.0
	0x7f7f73fb4000-0x7f7f73fb5000	
	0x7f7f73fb5000-0x7f7f73fb6000	/usr/lib/x86_64-linux-gnu/libxshmfence.so.1.0.0
	0x7f7f73fb6000-0x7f7f741b6000	/usr/lib/x86_64-linux-gnu/libxshmfence.so.1.0.0
	0x7f7f741b6000-0x7f7f741b7000	/usr/lib/x86_64-linux-gnu/libxshmfence.so.1.0.0
	0x7f7f741b7000-0x7f7f741b8000	/usr/lib/x86_64-linux-gnu/libxshmfence.so.1.0.0
	0x7f7f741b8000-0x7f7f741bd000	/usr/lib/x86_64-linux-gnu/libxcb-sync.so.1.0.0
	0x7f7f741bd000-0x7f7f743bd000	/usr/lib/x86_64-linux-gnu/libxcb-sync.so.1.0.0
	0x7f7f743bd000-0x7f7f743be000	/usr/lib/x86_64-linux-gnu/libxcb-sync.so.1.0.0
	0x7f7f743be000-0x7f7f743bf000	/usr/lib/x86_64-linux-gnu/libxcb-sync.so.1.0.0
	0x7f7f743bf000-0x7f7f743c1000	/usr/lib/x86_64-linux-gnu/libxcb-present.so.0.0.0
	0x7f7f743c1000-0x7f7f745c0000	/usr/lib/x86_64-linux-gnu/libxcb-present.so.0.0.0
	0x7f7f745c0000-0x7f7f745c1000	/usr/lib/x86_64-linux-gnu/libxcb-present.so.0.0.0
	0x7f7f745c1000-0x7f7f745c2000	/usr/lib/x86_64-linux-gnu/libxcb-present.so.0.0.0
	0x7f7f745c2000-0x7f7f745c4000	/usr/lib/x86_64-linux-gnu/libxcb-dri3.so.0.0.0
	0x7f7f745c4000-0x7f7f747c3000	/usr/lib/x86_64-linux-gnu/libxcb-dri3.so.0.0.0
	0x7f7f747c3000-0x7f7f747c4000	/usr/lib/x86_64-linux-gnu/libxcb-dri3.so.0.0.0
	0x7f7f747c4000-0x7f7f747c5000	/usr/lib/x86_64-linux-gnu/libxcb-dri3.so.0.0.0
	0x7f7f747c5000-0x7f7f747eb000	/lib/x86_64-linux-gnu/libexpat.so.1.6.0
	0x7f7f747eb000-0x7f7f749eb000	/lib/x86_64-linux-gnu/libexpat.so.1.6.0
	0x7f7f749eb000-0x7f7f749ed000	/lib/x86_64-linux-gnu/libexpat.so.1.6.0
	0x7f7f749ed000-0x7f7f749ee000	/lib/x86_64-linux-gnu/libexpat.so.1.6.0
	0x7f7f749ee000-0x7f7f74b60000	/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21
	0x7f7f74b60000-0x7f7f74d60000	/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21
	0x7f7f74d60000-0x7f7f74d6a000	/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21
	0x7f7f74d6a000-0x7f7f74d6c000	/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21
	0x7f7f74d6c000-0x7f7f74d70000	
	0x7f7f74d70000-0x7f7f74d94000	/lib/x86_64-linux-gnu/libpng12.so.0.54.0
	0x7f7f74d94000-0x7f7f74f93000	/lib/x86_64-linux-gnu/libpng12.so.0.54.0
	0x7f7f74f93000-0x7f7f74f94000	/lib/x86_64-linux-gnu/libpng12.so.0.54.0
	0x7f7f74f94000-0x7f7f74f95000	/lib/x86_64-linux-gnu/libpng12.so.0.54.0
	0x7f7f74f95000-0x7f7f74fb9000	/usr/lib/x86_64-linux-gnu/libjpeg.so.62.0.0
	0x7f7f74fb9000-0x7f7f751b8000	/usr/lib/x86_64-linux-gnu/libjpeg.so.62.0.0
	0x7f7f751b8000-0x7f7f751b9000	/usr/lib/x86_64-linux-gnu/libjpeg.so.62.0.0
	0x7f7f751b9000-0x7f7f751ba000	/usr/lib/x86_64-linux-gnu/libjpeg.so.62.0.0
	0x7f7f751ba000-0x7f7f753d5000	/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
	0x7f7f753d5000-0x7f7f755d4000	/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
	0x7f7f755d4000-0x7f7f755f0000	/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
	0x7f7f755f0000-0x7f7f755fc000	/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
	0x7f7f755fc000-0x7f7f755ff000	
	0x7f7f755ff000-0x7f7f7565d000	/lib/x86_64-linux-gnu/libssl.so.1.0.0
	0x7f7f7565d000-0x7f7f7585d000	/lib/x86_64-linux-gnu/libssl.so.1.0.0
	0x7f7f7585d000-0x7f7f75861000	/lib/x86_64-linux-gnu/libssl.so.1.0.0
	0x7f7f75861000-0x7f7f75868000	/lib/x86_64-linux-gnu/libssl.so.1.0.0
	0x7f7f75868000-0x7f7f75881000	/lib/x86_64-linux-gnu/libz.so.1.2.8
	0x7f7f75881000-0x7f7f75a80000	/lib/x86_64-linux-gnu/libz.so.1.2.8
	0x7f7f75a80000-0x7f7f75a81000	/lib/x86_64-linux-gnu/libz.so.1.2.8
	0x7f7f75a81000-0x7f7f75a82000	/lib/x86_64-linux-gnu/libz.so.1.2.8
	0x7f7f75a82000-0x7f7f75aef000	/usr/lib/x86_64-linux-gnu/libGLU.so.1.3.1
	0x7f7f75aef000-0x7f7f75cef000	/usr/lib/x86_64-linux-gnu/libGLU.so.1.3.1
	0x7f7f75cef000-0x7f7f75cf0000	/usr/lib/x86_64-linux-gnu/libGLU.so.1.3.1
	0x7f7f75cf0000-0x7f7f75cf1000	/usr/lib/x86_64-linux-gnu/libGLU.so.1.3.1
	0x7f7f75cf1000-0x7f7f75d61000	/usr/lib/x86_64-linux-gnu/mesa/libGL.so.1.2.0
	0x7f7f75d61000-0x7f7f75f60000	/usr/lib/x86_64-linux-gnu/mesa/libGL.so.1.2.0
	0x7f7f75f60000-0x7f7f75f63000	/usr/lib/x86_64-linux-gnu/mesa/libGL.so.1.2.0
	0x7f7f75f63000-0x7f7f75f64000	/usr/lib/x86_64-linux-gnu/mesa/libGL.so.1.2.0
	0x7f7f75f64000-0x7f7f75f65000	
	0x7f7f75f65000-0x7f7f75f7b000	/lib/x86_64-linux-gnu/libgcc_s.so.1
	0x7f7f75f7b000-0x7f7f7617a000	/lib/x86_64-linux-gnu/libgcc_s.so.1
	0x7f7f7617a000-0x7f7f7617b000	/lib/x86_64-linux-gnu/libgcc_s.so.1
	0x7f7f7617b000-0x7f7f76283000	/lib/x86_64-linux-gnu/libm-2.23.so
	0x7f7f76283000-0x7f7f76482000	/lib/x86_64-linux-gnu/libm-2.23.so
	0x7f7f76482000-0x7f7f76483000	/lib/x86_64-linux-gnu/libm-2.23.so
	0x7f7f76483000-0x7f7f76484000	/lib/x86_64-linux-gnu/libm-2.23.so
	0x7f7f76484000-0x7f7f76487000	/lib/x86_64-linux-gnu/libdl-2.23.so
	0x7f7f76487000-0x7f7f76686000	/lib/x86_64-linux-gnu/libdl-2.23.so
	0x7f7f76686000-0x7f7f76687000	/lib/x86_64-linux-gnu/libdl-2.23.so
	0x7f7f76687000-0x7f7f76688000	/lib/x86_64-linux-gnu/libdl-2.23.so
	0x7f7f76688000-0x7f7f766a0000	/lib/x86_64-linux-gnu/libpthread-2.23.so
	0x7f7f766a0000-0x7f7f7689f000	/lib/x86_64-linux-gnu/libpthread-2.23.so
	0x7f7f7689f000-0x7f7f768a0000	/lib/x86_64-linux-gnu/libpthread-2.23.so
	0x7f7f768a0000-0x7f7f768a1000	/lib/x86_64-linux-gnu/libpthread-2.23.so
	0x7f7f768a1000-0x7f7f768a5000	
	0x7f7f768a5000-0x7f7f76a65000	/lib/x86_64-linux-gnu/libc-2.23.so
	0x7f7f76a65000-0x7f7f76c65000	/lib/x86_64-linux-gnu/libc-2.23.so
	0x7f7f76c65000-0x7f7f76c69000	/lib/x86_64-linux-gnu/libc-2.23.so
	0x7f7f76c69000-0x7f7f76c6b000	/lib/x86_64-linux-gnu/libc-2.23.so
	0x7f7f76c6b000-0x7f7f76c6f000	
	0x7f7f76c6f000-0x7f7f76ca9000	/usr/lib/x86_64-linux-gnu/libubsan.so.0.0.0
	0x7f7f76ca9000-0x7f7f76ea8000	/usr/lib/x86_64-linux-gnu/libubsan.so.0.0.0
	0x7f7f76ea8000-0x7f7f76eaa000	/usr/lib/x86_64-linux-gnu/libubsan.so.0.0.0
	0x7f7f76eaa000-0x7f7f76eab000	/usr/lib/x86_64-linux-gnu/libubsan.so.0.0.0
	0x7f7f76eab000-0x7f7f7797c000	
	0x7f7f7797c000-0x7f7f7a77c000	/home/gpac/gpac_public-master/destdir/usr/local/lib/libgpac.so.8.0.0
	0x7f7f7a77c000-0x7f7f7a97b000	/home/gpac/gpac_public-master/destdir/usr/local/lib/libgpac.so.8.0.0
	0x7f7f7a97b000-0x7f7f7a984000	/home/gpac/gpac_public-master/destdir/usr/local/lib/libgpac.so.8.0.0
	0x7f7f7a984000-0x7f7f7c872000	/home/gpac/gpac_public-master/destdir/usr/local/lib/libgpac.so.8.0.0
	0x7f7f7c872000-0x7f7f7c874000	
	0x7f7f7c874000-0x7f7f7c968000	/usr/lib/x86_64-linux-gnu/libasan.so.2.0.0
	0x7f7f7c968000-0x7f7f7cb68000	/usr/lib/x86_64-linux-gnu/libasan.so.2.0.0
	0x7f7f7cb68000-0x7f7f7cb6b000	/usr/lib/x86_64-linux-gnu/libasan.so.2.0.0
	0x7f7f7cb6b000-0x7f7f7cb6c000	/usr/lib/x86_64-linux-gnu/libasan.so.2.0.0
	0x7f7f7cb6c000-0x7f7f7d7e1000	
	0x7f7f7d7e1000-0x7f7f7d807000	/lib/x86_64-linux-gnu/ld-2.23.so
	0x7f7f7d807000-0x7f7f7d9f9000	
	0x7f7f7d9f9000-0x7f7f7da06000	
	0x7f7f7da06000-0x7f7f7da07000	/lib/x86_64-linux-gnu/ld-2.23.so
	0x7f7f7da07000-0x7f7f7da08000	/lib/x86_64-linux-gnu/ld-2.23.so
	0x7f7f7da08000-0x7f7f7da09000	
	0x7fffa2741000-0x7fffa2762000	[stack]
	0x7fffa27ba000-0x7fffa27bd000	[vvar]
	0x7fffa27bd000-0x7fffa27bf000	[vdso]
	0xffffffffff600000-0xffffffffff601000	[vsyscall]
==4700==End of process memory map.
==4700==AddressSanitizer CHECK failed: ../../../../src/libsanitizer/sanitizer_common/sanitizer_posix.cc:121 "(("unable to mmap" && 0)) != (0)" (0x0, 0x0)
    #0 0x7f7f7c914631  (/usr/lib/x86_64-linux-gnu/libasan.so.2+0xa0631)
    #1 0x7f7f7c9195e3 in __sanitizer::CheckFailed(char const*, int, char const*, unsigned long long, unsigned long long) (/usr/lib/x86_64-linux-gnu/libasan.so.2+0xa55e3)
    #2 0x7f7f7c921611  (/usr/lib/x86_64-linux-gnu/libasan.so.2+0xad611)
    #3 0x7f7f7c896c0c  (/usr/lib/x86_64-linux-gnu/libasan.so.2+0x22c0c)
    #4 0x7f7f7c90c5d2 in malloc (/usr/lib/x86_64-linux-gnu/libasan.so.2+0x985d2)
    #5 0x7f7f7984da77 in metx_Read (destdir/usr/local/lib/libgpac.so.8+0x1ed1a77)
    #6 0x7f7f798c4235 in gf_isom_box_parse_ex.constprop.5 (destdir/usr/local/lib/libgpac.so.8+0x1f48235)
    #7 0x7f7f79a6e336 in abst_Read (destdir/usr/local/lib/libgpac.so.8+0x20f2336)
    #8 0x7f7f798c4235 in gf_isom_box_parse_ex.constprop.5 (destdir/usr/local/lib/libgpac.so.8+0x1f48235)
    #9 0x7f7f798c5034 in gf_isom_parse_root_box (destdir/usr/local/lib/libgpac.so.8+0x1f49034)
    #10 0x7f7f798efa1f in gf_isom_parse_movie_boxes (destdir/usr/local/lib/libgpac.so.8+0x1f73a1f)
    #11 0x7f7f798f52c8 in gf_isom_open_file (destdir/usr/local/lib/libgpac.so.8+0x1f792c8)
    #12 0x42f469 in mp4boxMain /home/gpac/gpac_public-master/applications/mp4box/main.c:4818
    #13 0x7f7f768c582f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2082f)
    #14 0x410558 in _start (/home/gpac/gpac_public-master/destdir/usr/local/bin/MP4Box+0x410558)
```

**Comment by cpviolator** [comment]
Just hit a similar issue. Notice that the byte size is greater than the size of unsigned long int:

`4,294,975,488 > 4,294,967,295` So it's likely that AddressSanitiser is limited by this.

BTW, I'm allocated memory using `unsigned long long int`.

**Comment by jeanlf** [comment]
This has been fixed in bd96f67, thanks for the report
